### ISSUE TITLE
Too narrow code block

### ISSUE THREAD

**Issue by Korb** [alexwforsythe/code-blocks#255]
When using pageless page setup, the code block takes up a small portion of the document width. I would prefer to see the code block always at the full width of the document. As a last resort, if the width must be specified not as a percentage, but in units of length, I would like to set the width of the created block myself, which would suit me better than the current one. And so that from now on all formatted code blocks will be created with this width.

**Comment by weizlini** [comment]
yes i second this! extremely annoying. constantly requires me to reformat exported code with line numbers because they are all wrapping
